measure/src/main/scala/org/apache/griffin/measure/step/builder/dsl/transform/Expr2DQSteps.scala

```diff
@@ -32,6 +33,10 @@ trait Expr2DQSteps extends Loggable with Serializable {
 
   def getDQSteps(): Seq[DQStep]
 
+  implicit def recoredWriteStepToList(writeStep: RecordWriteStep): List[WriteStep] = writeStep :: Nil
+
+  implicit def metricsWriteStepToList(writeStep: MetricWriteStep): List[WriteStep] = writeStep :: Nil
```

Comment:
It's a little weird to implicitly change a `WriteStep` into `List[WriteStep]`, it would be clearer and easier for reading if we just pass the List parameter as the function defined.

Reply:
I also think implicitly convert WriteStep into List[WriteStep] is over-assumption. we should explicitly declare it when used. 
@wankunde What is your opinion?
If we have to change the interface, then we can change it to make dq chain more clear.

Reply:
@bhlx3lyx7 @guoyuepeng 
At first , I think for a TransformStep there could be one sink or a sink list. So I add a implicit method，and the argument can be WriteStep or List[WriteStep].

But now,I think for the result table or dataframe of a transformStep, there could be only one WriteStep( one sink can sink to different places  and we don't need different output logic). So  i want to chuange the interface to **Option[WriteStep]** .

Is that ok ?

Reply:
make sense to me .